docker/dockerfiles/binary_injected.Dockerfile

```diff
@@ -8,22 +8,22 @@ ARG BUILD_DATE
 ARG IMAGE_NAME
 
 # That can be a single one or a comma separated list
-ARG BINARY=nh-node
+ARG BINARY=zkv-node
 
 ARG BIN_FOLDER=.
 ARG DOC_URL=https://github.com/HorizenLabs/zkVerify
 ARG DESCRIPTION="zkVerify"
 ARG AUTHORS="mainchain-team@horizenlabs.io"
 ARG VENDOR="Horizen Labs"
 
-LABEL io.hl.image.authors=${AUTHORS} \
-	io.hl.image.vendor="${VENDOR}" \
-	io.hl.image.revision="${VCS_REF}" \
-	io.hl.image.title="${IMAGE_NAME}" \
-	io.hl.image.created="${BUILD_DATE}" \
-	io.hl.image.documentation="${DOC_URL}" \
-	io.hl.image.description="${DESCRIPTION}" \
-	io.hl.image.source="https://github.com/HorizenLabs/zkVerify/blob/${VCS_REF}/docker/dockerfiles/binary_injected.Dockerfile"
```

Comment:
I'm uncertain on these renamings `hl` -> `zkv`. What do you guys think?

Reply:
Here I prefer hl instead. It's the vendor and not the product.

Reply:
Ok makes sense for these variables, thanks.

I'd leave the renaming of the docker files though, since that is indeed the product.